README.md

```diff
@@ -6,7 +6,11 @@
 
 Build all targets:
 
-```make```
+```
+git submodule init
+git submodule update
```

Comment:
@potaito - 
I would add the init and recursive like this:
`git submodule sync --recursive && git submodule update --init --recursive`

Reply:
Thanks, I made the modifications you suggested. 

I blatantly stole that from https://dev.px4.io/en/software_update/stm32_bootloader.html#building-the-bootloader